libs/core-ui/.eslintrc.json

```diff
@@ -7,7 +7,11 @@
       "rules": {}
     },
     {
-      "files": ["JointDataset.ts"],
+      "files": [
+        "JointDataset.ts",
+        "CohortEditorFilterSection.tsx",
```

Comment:
why do these two need to be ignored for the lint ?

Reply:
Otherwise I can try to refactor them.
Adding them here is temporary as they will be back to less than 30 lines after removing the old code